src/Components/Shifting/ListFilter.tsx

```diff
@@ -69,7 +69,7 @@ export default function ListFilter(props: any) {
       if (filter.orgin_facility) {
         setOriginLoading(true);
         const res = await dispatch(
-          getFacility(filter.orgin_facility, "orgin_facility")
+          getAllFacility(filter.orgin_facility, "orgin_facility")
         );
         if (res && res.data) {
           setFilterState({ orgin_facility_ref: res.data });
```

Comment:
There are repeatition of useEffect code for each filter. Will have to look into it for next task. 😅